app/services/mapping_code_determiner.rb

```diff
@@ -53,8 +53,14 @@ def dispute_type_mapping_code
   def case_type_mapping_code
     # TODO: Add further when-branches once we have additional case types
     case tribunal_case.case_type
+    when CaseType::APPLY_DECISION_ENQUIRY
+      MappingCode::APPN_DECISION_ENQRY
```

Comment:
`APPN` is application, as opposed to `APPEAL` which is... errr, that.

The constants are the verbatim mapping codes from GLiMR, which are like that because of the 20 character limit it has on mapping codes.